lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
```

Comment:
I'm kind of wonderering if we should go with `VO` or even `ValueObject` instead of `Popo`. I got what it meant because of Pojo, but I'm not certain how well known that terminology is in PHP... What do you think?